src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -69,10 +68,7 @@ public async Task<Connection> ActivateAsync(CancellationToken token)
         {
             var (clientStream, serverStream) = FullDuplexStream.CreatePair();
 
-            // Need an auto-flushing stream for the server because O# doesn't currently flush after writing responses. Without this
-            // performing the Initialize handshake with the LanguageServer hangs.
-            var autoFlushingStream = new AutoFlushingStream(serverStream);
```

Comment:
I actually think we might still need this because O# calls Flush (not FlushAsync) after writes. Could you try writing a TagHelper attribute (via completions) to ensure this doesn't explode?

Reply:
As discussed offline, this scenario actually explodes with or without the `AutoFlushingStream`.